ui/src/templates/forms/region.hbs

```diff
@@ -7,21 +7,21 @@
         <form id="regionFormEdit" class="XiboForm">
             <div class="tab-content">
                 <div class="tab-pane active" id="generalTab">
-                    {{> inputs/hidden id="type" value=region.type }}
-                    {{> inputs/text value=region.name id="name" title="Name" helpText="An optional name for this widget" }}
-                    {{> inputs/checkbox value=(getOption region.regionOptions "loop") id="loop" title="Loop?" helpText="If there is only one item in this region should it loop?" }}
+                    {{> inputs/hidden id="type" value=region.type customClass="position-input" }}
+                    {{> inputs/text value=region.name id="name" title="Name" helpText="An optional name for this widget" customClass="position-input" }}
```

Comment:
We have a bunch of stuff here that isn't translated, so we need to have a think about how this gets handled (not for this PR, but we can open an issue for it please)

Reply:
I've created a new issue for this